src/ape/utils/abi.py

```diff
@@ -99,6 +99,7 @@ def parse(self, output_types: List[ABIType], values: Union[List, Tuple]) -> Any:
         else:
             for output_type, value in zip(output_types, values):
                 if isinstance(value, (tuple, list)):
+                    output_type = output_type.copy()
```

Comment:
Does this line work? Or is it basically trying not to overwrite an input variable here would rather have a new variable if that's the case

Reply:
It should, my problem does not hit it though. Need another example.